pages/about.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/mobile-screen.css">
+    <link rel="stylesheet" href="../css/about-mobile-screen.css">
+    <link rel="stylesheet" media="screen and (min-width: 768px)" href="../css/wide-screen.css">
+    <link rel="stylesheet" media="screen and (min-width: 768px)" href="../css/about-wide-screen.css">
+    <title>About</title>
+</head>
+
+<body>
+    <section class="header-background">
+        <header class="header">
+            <div class="social-media">
+                <a href="#" class="social-media-link">
+                    <img src="" alt="">
+                </a>
+                <a href="" class="social-media-link">
+                    <img src="" alt="">
+                </a>
+                <a href="" class="last" class="social-media-link"> English </a>
+            </div>
+            <nav class="nav-bar">
+                <a href="../index.html" class="logo"> <img src="../img/logo_muda.jpg" alt="" width="80px" height="70px"></a>
+                <ul class="nav-menu">
+                    <li class="nav-item nav-item-1">
+                        <a href="about.html" class="nav-link"> About</a>
+                    </li>
```

Comment:
- [ ] It is redundant to have an `àbout` link on the about page. I kindly suggest you to replace this `about` page link with the `home` page link.
![Capture](https://user-images.githubusercontent.com/93598189/197206017-80109c61-b11a-447f-b010-6968e81938aa.PNG)

- [ ] For  hamburger menu for about page please also try to add home page link .

![Capture](https://user-images.githubusercontent.com/93598189/197210057-bc361305-1f13-4bb4-9480-6eed263db51c.PNG)